seqr/views/utils/export_utils.py

```diff
@@ -52,21 +55,21 @@ def export_table(filename_prefix, header, rows, file_format, titlecase_header=Tr
         response = HttpResponse(content_type='text/tsv')
         response['Content-Disposition'] = 'attachment; filename="{}.tsv"'.format(filename_prefix)
         response.writelines(['\t'.join(header)+'\n'])
-        response.writelines(('\t'.join(map(unicode, row))+'\n' for row in rows))
+        response.writelines('\t'.join(list(map(str, row)))+'\n' for row in rows)
```

Comment:
we explicitly had code to make sure you could export unicode and this takes out that functionality and restricts it to strings. Why? With unicode literals we could just take out the `map` entirely

Reply:
The `row` is a list of some different types include `str`, `int` and maybe others. The `map()` converts all the members (especially non-str members) in the list into `str`.
The `str` here is imported from `builtins` which means it behaves the same as the `str` in Python 3 and it is unicode only.